components/Order/basic_usage.rst

```diff
@@ -0,0 +1,251 @@
+Basic Usage
+===========
+
+Adjustments
+-----------
+
+Neutral Adjustments
+~~~~~~~~~~~~~~~~~~~
+
+In some cases, you may want to use **Adjustment** just for displaying purposes.
+For example, when your order items have the tax already included in the price.
+
+Every **Adjustment** instance has the ``neutral`` property, which indicates if it should be counted against object total.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Order;
+    use Sylius\Component\Order\OrderItem;
+    use Sylius\Component\Order\Adjustment;
+
+    $order = new Order();
+    $tshirt = new OrderItem();
+    $tshirt->setUnitPrice(4999);
+
+    $shippingFees = new Adjustment();
+    $shippingFees->setAmount(1000);
+
+    $tax = new Adjustment();
+    $tax->setAmount(1150);
+    $tax->setNeutral(true);
+
+    $order->addItem($tshirt);
+    $order->addAdjustment($shippingFees);
+    $order->addAdjustment($tax);
+
+    $order->calculateTotal();
+    $order->getTotal();
+
+    // Output will be 5999.
```

Comment:
I would suggest keeping the comments inline with method execution
```php
$order->getTotal(); // Output will be 5999
```

Here also we could explain the 5999 value that it is 4999 - tshirt price with included tax and 1000 - shipping fee